collectors/0/druid.py

```diff
@@ -1,123 +1,83 @@
-        'request/time':                     {'user2', 'user4', 'user6'},
-        'server/segment/used':              {'user1', 'user2'},
-        'server/segment/usedPercent':       {'user1', 'user2'},
-        'server/segment/count':             {'user1', 'user2'},
-        'server/segment/totalUsed':         {'user2'},
-        'server/segment/totalUsedPercent':  {'user2'},
-        'server/segment/totalCount':        {'user2'},
-        'events/thrownAway':                {'user2'},
-        'events/unparseable':               {'user2'},
-        'events/processed':                 {'user2'},
-        'rows/output':                      {'user2'},
-        'coordinator/segment/size':         {'user1'},
-        'coordinator/segment/count':        {'user1'},
-        'coordinator/loadQueue/size':       {'user1'},
-        'coordinator/loadQueue/failed':     {'user1'},
-        'coordinator/loadQueue/count':      {'user1'},
-        'coordinator/dropQueue/count':      {'user1'},
-        'indexer/time/run/millis':          {'user2', 'user3', 'user4'},
-        'indexer/segment/bytes':            {'user2', 'user4'},
-        'indexer/segmentMoved/bytes':       {'user2', 'user4'},
-        'indexer/segmentNuked/bytes':       {'user2', 'user4'}
-        }
-
-
-def report(line):
-    match = EVENT_REGEX.match(line.strip())
-
-    if not match:
-        return
-    
-    event = json.loads(match.group(1))
-
-    druid_metric = event['metric']
-    metric = 'druid.' + druid_metric.replace('/', '.')
-    value = event['value']
-    timestamp = calendar.timegm(dtparser.parse(event['timestamp']).utctimetuple())
-
-    tags_to_include = COMMON_TAGS
-    metric_specific_tags = METRIC_SPECIFIC_TAGS.get(druid_metric)
-    if metric_specific_tags:
-        tags_to_include = tags_to_include | metric_specific_tags
-
-    tags = { k: v for k, v in event.iteritems() if k in tags_to_include }
-
-    print format_tsd_key(metric, value, timestamp, tags)
-
- 
-def spawn_monitor(metric_file):
-    name = 'monitor for ' + metric_file
+    'coordinator',
+    'overlord',
+    'historical',
+    'broker',
+    'realtime'
+]
+
 
-    monitor = subprocess.Popen(
-            ['/usr/bin/tail', '--lines=0', '--follow=name', '--retry', metric_file],
-            bufsize=1,  # line buffered
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE
-            )
+def get_metric_file(role):
+    return DRUID_METRICS_DIR + '/' + role + '.log'
+
+
+def spawn_monitor(role):
+    metric_file = get_metric_file(role)
+    name = 'monitor for ' + metric_file
 
-    # ignore stderr
-    monitor.stderr.close()
+    tailor = subprocess.Popen(
+        ['tail', '--lines=0', '--follow=name', '--retry', metric_file],
+        bufsize=1,  # line buffered
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
+    )
+
+    aggregator = subprocess.Popen(
+        ['java', '-jar', SCRIPT_DIR + '/../../collectors/lib/druid-metric-aggregator-0.0.15-all.jar',
```

Comment:
Is there some way we can specify the path to the jar via a config? Otherwise, on every update we'll have to make a code change.